syncstorage/src/settings.rs

```diff
@@ -289,6 +289,7 @@ impl Settings {
         s.set_default("tokenserver.node_type", "spanner")?;
         s.set_default("tokenserver.statsd_label", "syncstorage.tokenserver")?;
         s.set_default("tokenserver.run_migrations", cfg!(test))?;
+        s.set_default("tokenserver.token_duration", 3600)?;
```

Comment:
The Python Tokenserver has a default duration of 300 seconds, but it has the duration set to 3600 in the settings: https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/tokenserver/puppet/modules/tokenserver/templates/settings.ini.erb#L55